Makefile

```diff
@@ -11,34 +11,25 @@ BIN_DIR = $(CURDIR)/build/_output/bin/
 
 export GOFLAGS=-mod=vendor
 export GO111MODULE=on
-export GOROOT=$(BIN_DIR)/go/
-export GOBIN=$(GOROOT)/bin/
-export PATH := $(GOBIN):$(PATH)
-export GO := $(GOBIN)/go
-KUSTOMIZE := GOFLAGS=-mod=mod $(GO) run sigs.k8s.io/kustomize/kustomize/v4@v4.5.2
-CONTROLLER_GEN := GOFLAGS=-mod=mod $(GO) run sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.0
-GOFMT := GOFLAGS=-mod=mod $(GO)fmt
-VET := GOFLAGS=-mod=mod $(GO) vet
-DEEPCOPY_GEN := GOFLAGS=-mod=mod $(GO) install k8s.io/code-generator/cmd/deepcopy-gen@latest
+KUSTOMIZE := GOFLAGS=-mod=mod go run sigs.k8s.io/kustomize/kustomize/v4@v4.5.2
+CONTROLLER_GEN := GOFLAGS=-mod=mod go run sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.0
+DEEPCOPY_GEN := GOFLAGS=-mod=mod go install k8s.io/code-generator/cmd/deepcopy-gen@latest
 GO_VERSION = $(shell hack/go-version.sh)
 
 E2E_TEST_EXTRA_ARGS ?=
 export E2E_TEST_TIMEOUT ?= 1h
-E2E_TEST_ARGS ?= $(strip -test.v -test.timeout=$(E2E_TEST_TIMEOUT) -ginkgo.timeout=$(E2E_TEST_TIMEOUT) -ginkgo.v $(E2E_TEST_EXTRA_ARGS))
+E2E_TEST_ARGS ?= $(strip -test.v -test.timeout=$(E2E_TEST_TIMEOUT) -ginkgo.timeout=$(E2E_TEST_TIMEOUT) $(E2E_TEST_EXTRA_ARGS))
 
 export KUBECTL ?= cluster/kubectl.sh
 
 all: generate
 
-$(GO):
-	hack/install-go.sh $(BIN_DIR) > /dev/null
-
 # Run tests
-test: $(GO)
-	$(GO) test ./pkg/... ./cmd/... -coverprofile cover.out
+test:
+	go test ./pkg/... ./cmd/... -coverprofile cover.out
```

Comment:
doesnt it means that running it locally might override the go version that the user already have ?
it isn't good if so